bin/commands/init/generate/cli.ts

```diff
@@ -0,0 +1,20 @@
+/*
+  This program and the accompanying materials are made available
+  under the terms of the Eclipse Public License v2.0 which
+  accompanies this distribution, and is available at
+  https://www.eclipse.org/legal/epl-v20.html
+ 
+  SPDX-License-Identifier: EPL-2.0
+ 
+  Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'cm_std';
+import * as index from './index';
+import * as configmgr from '../../../libs/configmgr';
+import * as common from '../../../libs/common';
+
+index.execute(!!std.getenv('ZWE_CLI_PARAMETER_DRY_RUN'));
```

Comment:
`zwe init generate -c zowe.yaml --dry-run` will always run a job.

The definition in [`/bin/init/.parameters` -> `security-dry-run,dry-run`](https://github.com/zowe/zowe-install-packaging/blob/07117559724a383b8155bf279a55a61fecddf0d0/bin/commands/init/.parameters#L3) and how `zwe` evaluates flags will cause:
* Always nothing in `ZWE_CLI_PARAMETER_DRY_RUN`
* Boolean only here: `ZWE_CLI_PARAMETER_SECURITY_DRY_RUN=true|false`
* `dry-run` is shortcut, but only main variable is set

This should be probably:
`index.execute(!!std.getenv('ZWE_CLI_PARAMETER_SECURITY_DRY_RUN'));`

Or for the future:

`index.execute(!!std.getenv('ZWE_CLI_PARAMETER_DRY_RUN') || !!std.getenv('ZWE_CLI_PARAMETER_SECURITY_DRY_RUN'));`

Reply:
https://github.com/zowe/zowe-install-packaging/pull/3718/commits/29d19b8c8d60c6d96dfcb7df10a597b29cc13e09